VRDR/DeathRecord.cs

```diff
@@ -10161,15 +10144,20 @@ private void RestoreReferences()
             }
 
             // Grab Certifier
-            if (fullRecord && (Composition.Attester == null || Composition.Attester.FirstOrDefault() == null || Composition.Attester.First().Party == null || String.IsNullOrWhiteSpace(Composition.Attester.First().Party.Reference)))
-            {
-                throw new System.ArgumentException("The Composition is missing an attestor (a reference to the Certifier/Practitioner resource).");
-            }
-            var practitionerEntry = Bundle.Entry.FirstOrDefault(entry => entry.Resource.ResourceType == ResourceType.Practitioner && (entry.FullUrl == Composition.Attester.First().Party.Reference ||
-            (entry.Resource.Id != null && entry.Resource.Id == Composition.Attester.First().Party.Reference)));
-            if (practitionerEntry != null)
+            if ( Composition == null || (Composition.Attester == null || Composition.Attester.FirstOrDefault() == null || Composition.Attester.First().Party == null || String.IsNullOrWhiteSpace(Composition.Attester.First().Party.Reference)))
             {
-                Certifier = (Practitioner)practitionerEntry.Resource;
+                if(fullRecord)
+                {
+                    throw new System.ArgumentException("The Composition is missing an attestor (a reference to the Certifier/Practitioner resource).");
+                }
+            }else{  // There is an attester
```

Comment:
Need to use correct formatting (brackets on their own line)

Reply:
(Just run the formatter)

Reply:
option-shift-f, option-shift-f, option-shift-f.... my new mantra.